### ISSUE TITLE
Allow users to opt out of promotional emails

### ISSUE THREAD

**Issue by DougCoffland** [buildbotics/buildbotics-ui#45]
We probably need to give users the ability to opt out of promotional materials and advertising when they register.  I am including this option in our Privacy Policy.

**Comment by jcoffland** [comment]
This should probably be reversed.  We should allow users to opt-in to our mailing list.

**Comment by DougCoffland** [comment]
I agree.  I didn't realize we were adding them to our mailing list.  How do
we change it?

Doug

**Comment by jcoffland** [comment]
We're not.  This ticket suggests that we would.
